jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -246,15 +266,61 @@ def _on_rs_redshift_updated(self, event):
         self._rs_disable_observe = True
         self.rs_rv = self._redshift_to_velocity(value)
         self._rs_disable_observe = False
-
         self._update_line_positions()
 
         if not self._rs_pause_tables:
+            # TODO: try to avoid essentially repeating the loop from above, careful to minimize
+            # updates to vue, maybe pause traitlets?
+            new_list_contents = {}
+            for list_name, line_list in self.list_contents.items():
+                for i, line in enumerate(line_list['lines']):
+                    if self._rs_line_obs_change[0] == list_name and self._rs_line_obs_change[1] == i:  # noqa
+                        # this trigger is coming from a manual change to the observed
+                        # wavelength and would result in a small change to the value before the
+                        # user can finish typing.  So we'll just keep the old value until the
+                        # widget is blurred (loses focus)
+                        line_list['lines'][i]['obs'] = self._rs_line_obs_change[2]
+                    else:
+                        line_list['lines'][i]['obs'] = self._rest_to_obs(float(line['rest']))
+
+                new_list_contents[list_name] = line_list
+
+            self.list_contents = {}  # TODO: fix with dict magic thing
```

Comment:
Maybe explain more what you mean by "dict magic thing"? 😆

Reply:
I intentionally read through diffs before posting ready for review to make sure these don't slip through 😂 I'll take care of this one tomorrow too, thanks for catching it!